src/util/Backtrace.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "util/Backtrace.h"
+#include "config.h"
+
+#if HAVE_LIBUNWIND
```

Comment:
Minor question, but do we want `#if` or `#ifdef` here? Most of our configuration stuff is done with `#ifdef` but perhaps something has worked out differently here? Mostly, this is me being curious.

Reply:
In this case it won't make any difference but sure let's do ifdef